### ISSUE TITLE
Search Process: Raid.exe not found + Inject DDL in Raid.exe : Failed

### ISSUE THREAD

**Issue by audy228** [FarbstoffRSL/RSL-Helper#20]
I've tried to setup the path but it still doesn't work. No idea the reason... And it can't memorize the path i set.
Also, it shows "Inject DDL in Raid.exe : Failed ". Got no idea what the prob is...

**Comment by Dwza** [comment]
have you tried to deinstall helper, than totaly close plarium client (even from taskbar) than reinstall helper, start helper and than start plarium? and of course you have to start plarium client to get raid.exe, not blue stacks or any other emulators.

**Comment by audy228** [comment]
> have you tried to deinstall helper, than totaly close plarium client (even from taskbar) than reinstall helper, start helper and than start plarium? and of course you have to start plarium client to get raid.exe, not blue stacks or any other emulators.

So I do exactly what u replied above, but it still shows "Inject DDL in Raid.exe : Failed" and I suppose the main reason is that the traffic always shows 0MB below.

**Comment by Dwza** [comment]
have you started the tool as admin? and are you a "real" admin on your pc ?

**Comment by mooseontheloose89** [comment]
Hey @audy228 I had the exact same issue with the updated version, which I uninstalled and reinstalled the previous version without any luck..

I then uninstalled, reinstalled, ran as admin without luck either...

Uninstalled again, and when to the file locations of the RSLHelper and deleted the folder and all contents.

Reinstalled the 1.7v ran as admin and it's opened successfully.

Hope this helps! :)
